core/src/main/java/org/infinispan/marshall/exts/SecurityActions.java

```diff
@@ -24,39 +22,18 @@ private static Field getDeclaredField(Class<?> c, String fieldName) {
    }
 
    static Field getField(Class<?> c, String fieldName) {
```

Comment:
probably another candidate to be moved to `org.infinispan.commons.util.ReflectionUtil`